theme/src/components/widgets/spotify/media-item-grid.js

```diff
@@ -24,8 +24,16 @@ const placeholders = Array(12)
     </div>
   ))
 
-const MediaItemGrid = ({ isLoading, items = [] }) => {
+const MediaItemGrid = ({ isLoading, items = [], onTrackClick }) => {
   const [currentMediaId, setCurrentMediaId] = useState(false)
+
+  const handleClick = (e, spotifyURL) => {
+    e.preventDefault()
+    if (onTrackClick) {
```

Comment:
If onTrackClick isn't provided, the href url for these elements is still valid. Let's fall back to using that hyperlink and move `.preventDefault()` into this condition. If `onTrackClick` isn't provided, I'll exit early.